### PR TITLE
Update fundamentals 1

### PR DESCRIPTION
**Pull request by ethanwhite**
* Add new functions + if statements exercise that builds on dinosaur theme
* Add new DNA or RNA exercise without iteration (keeps old exercise with rename, but doesn't use)
* Minor cleanup

### CODE REVIEW

**Comment by KristinaRiemer** [comment]
Link to Nested Functions exercise at the end of the [functions lecture](http://datacarpentry.org/semester-biology/materials/functions-R/) is now broken due to name change.

File: exercises/Making-choices-dna-or-rna-R.md [hunk 1]
```diff
@@ -5,20 +5,19 @@ title: DNA or RNA
...
-sequence provided. Since all the function will know about the material is the
-sequence the only way to tell the difference between DNA and RNA is that
-RNA has the base Uracil (`"u"`) instead of the base Thymine (`"t"`). Have the
-function return one of three outputs: `"DNA"`, `"RNA"`, or `"UNKNOWN"`.
-
-1. Use the function and a `for` loop to print the type of the sequences in the
-following list.
-2. Use the function and `sapply` to print the type of the sequences in the
-following list.
+Write a function, `dna_or_rna(sequence)`, that determines if a sequence of base
```
Comment (KristinaRiemer) [inline exercises/Making-choices-dna-or-rna-R.md]: I think that the `dna_or_rna(sequence)` might be a little confusing to them. Maybe change the language to "Write a function called `dna_or_rna`, which takes a sequence as an argument, that determines..."?

**Comment by KristinaRiemer** [comment]
Maybe include the solution (i.e., the three expected mass values in order) for Size Estimates by Group exercise?

**Comment by ethanwhite** [comment]
Thanks @KristinaRiemer! All great suggestions. Fixed in new commit.

File: materials/functions-R.md [hunk 2]
```diff
@@ -158,4 +158,4 @@ est_shrub_mass_dim <- function(length, width, height){
 est_shrub_mass_dim(0.8, 1.6, 2.0)
 ```
 
-> Do [Nested Functions]({{ site.baseurl }}/exercises/Functions-nested-functions-R).
+> Do [Combined Functions]({{ site.baseurl }}/exercises/Functions-combined-functions-R).
```
Comment (KristinaRiemer) [inline materials/functions-R.md]: This link is still broken, it's supposed to be Functions-combining-functions-R, I think?

Comment (ethanwhite) [reply materials/functions-R.md]: Argh. Sorry. I've been having some git issues and apparently I didn't refix that in the mess. I'm going to go ahead and merge and then update shortly (for reasons related to the ongoing git issues).

**Comment by KristinaRiemer** [comment]
Everything else on this looked great.

**Pull request merged**